examples/data/ObsSet_example.cpp

```diff
@@ -30,23 +32,23 @@ int main(){
     std::vector<size_t> indices;
     indices.push_back(1);
     indices.push_back(2);
-    DataRepresentation dataRep2(indices);
+    ObsSet dataRep2(indices);
     // will look at observables at indices 1 and 2
 
     // ROOTNtuple nt("myfilename.root", "mytreename");
     std::vector<std::string> interestingObservablesForMyObject;
     interestingObservablesForMyObject.push_back("ob1");
     interestingObservablesForMyObject.push_back("ob2");
 
-    // DataRepresentation dataRepForMyObject = nt.MakeDataRep(interestingObservablesForMyObject);
+    // ObsSet dataRepForMyObject = nt.MakeObsSet(interestingObservablesForMyObject);
```

Comment:
I think MakeObsSet does not exist - in this case it was left as MakeDataRep, which is unfortunate, but oh well.

Reply:
Ah yes, you're right!